app/routes/task_routes.py

```diff
@@ -0,0 +1,109 @@
+from flask import Blueprint, request, make_response, jsonify, abort
+from app.models.task import Task
+from app import db
+from .routes_helpers import validate_model, delete_message
+from datetime import date
+import requests
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
```

Comment:
Since you're loading the `.env` file in the `__init__.py` file, there's no need to also load it again here. You could set the `SLACKBOT_TOKEN` in the `__init__.py` file and import it in this file to use.